archimedes/archimedes/runner.py

```diff
@@ -329,8 +329,8 @@ def main():
 
     request: RunRequest = RunRequest(
         isolator=args.isolator,
-        input_files=[make_storage_file(s) for s in args.inputs],
-        output_files=[make_storage_file(s) for s in args.outputs],
+        input_files=[make_storage_file(s) for s in args.inputs] if args.inputs else [],
```

Comment:
This works but is not robust -- you are forced to provide thi default anywhere args.input or arg.outputs is referenced.
It should be possible to set a default value to the arg parser to be an empty list when no options are set on the command line.